src/moments.jl

```diff
@@ -363,3 +363,44 @@ end
 
 kurtosis(v::RealArray) = kurtosis(v, mean(v))
 kurtosis(v::RealArray, wv::AbstractWeights) = kurtosis(v, wv, mean(v, wv))
+
+"""
+    cumulant(v, k, [wv::AbstractWeights], m=mean(v))
+
+Return the `k`th order cumulant of a real-valued array `v`, optionally
+specifying a weighting vector `wv` and a center `m`. This quantity is calculated using a recursive definition on lower order cumulants and central moments.
+"""
+function cumulant(v::RealArray, k::Int, m::Real)
+    cmoms = zeros(Float64, k)
+    cumls = zeros(Float64, k)
```

Comment:
Shouldn't hardcode `Float64` here:
```suggestion
    cmoms = zeros(typeof(m), k)
    cumls = zeros(typeof(m), k)
```
Could you also add a test with a non-`Float64` input type (e.g. `Int`)?